octoprint_mrbeam/templates/review.jinja2

```diff
@@ -15,23 +16,47 @@
                     <button value="8">8</button>
                     <button value="9">9</button>
                     <button value="10">10</button>
+                    <span class="rating-label">Extremely likely</span>
                 </div>
             </div>
+            <div id="review_thank_you" class="review_hidden_part">
+                <h3 class="review_title">{{ _('Thank you for your response.') }}</h3>
+            </div>
             <div id="review_how_can_we_improve" class="form-group review_hidden_part">
-                <h3 class="review_title">{{ _('Thank you!') }}</h3>
-                <label for="review_textarea" class="review_explanation">{{ _('Thank you for the feedback, is there anything else you want to tell us? Please share your thoughts with us so we can keep improving:') }}</label>
+                <label for="review_textarea" class="review_explanation">{{ _('Help us by explaining your score...') }}</label>
                 <textarea class="form-control" id="review_textarea" rows="4"></textarea>
             </div>
-            <div id="review_thank_you" class="review_hidden_part">
-                <h3 class="review_title">{{ _('Wow, thank you!') }}</h3>
-                <p class="review_explanation">{{ _('We love getting feedback from our customers! Do you have three minutes to leave a review for us? We’d be so excited to read it!') }}
-                    <a id="give_review_link" target="_blank" href="https://store.mr-beam.org/products/mr-beam-ii-dreamcut-air-filter-ii-bundle?utm_source=beamos&utm_medium=beamos&utm_campaign=review_screen#shopify-product-reviews">{{ _('Give review') }}</a>
+            <div id="ask_user_details" class="review_hidden_part">
+                <label for="review_input_phone_email">{{ _('Phone or email: (optional)') }}
+                    <small>{{ _('If it\'s ok to contact you about your feedback.') }}</small>
+                </label>
+                <input type="text" class="form-control" id="review_input_phone_email">
+            </div>
+            <div id="positive_review" class="review_hidden_part">
+                <h3 class="review_title">{{ _('Thank you for your response.') }}</h3>
+                <p>{{ _('Positive reviews from awesome customers like you help others to feel confident about choosing Mr Beam too.') }}</p>
+                <p>{{ _('Could you take 60 seconds to go to the ') }}
```

Comment:
Please bring this whole sentence in one translatable entity with placeholders for opening and closing tag. This is This is important for translations.